test/torchaudio_unittest/backend/sox_io/save_test.py

```diff
@@ -20,6 +21,9 @@
     name_func,
     get_enc_params,
 )
+from torch import (
+    zeros,
+)
```

Comment:
1. Please use `import torch` and `torch.zeros`.
2. This import should be placed before `from torchaudio.backend import sox_io_backend`, following PEP8.

Reply:
made this update